app/src/main/res/values/strings.xml

```diff
@@ -118,33 +118,35 @@
     <string name="notification_updatingArticles_full">Performing full update</string>
     <string name="notification_updatingArticles_fast">Performing fast update</string>
     <string name="notification_sweepingDeletedArticles">Sweeping deleted articles</string>
-    <string name="notification_downloadingImages">Downloading images</string>
-    <string name="notification_syncingQueue">Synchronizing local changes</string>
+    <string name="notification_downloadingImages">Downloading images…</string>
+    <string name="notification_syncingQueue">Syncing local changes…</string>
     <string name="notification_backgroundOperations">Background operations</string>
     <string name="notification_error">Error</string>
-    <string name="notification_incorrectCredentials">Incorrect credentials</string>
+    <string name="notification_incorrectCredentials">Wrong username or password</string>
```

Comment:
Anything else possible here?

Reply:
IIRC, incorrect `client ID` and `client secret` may cause that too. Maybe also things that cause HTTP 401 (like additional HTTP Basic auth), can't remember at the moment.

Reply:
If that happens to also be a thing to keep in mind, it can be added if it turns out to be a problem.

Reply:
What about "Wrong login data" or "Wrong credentials"?